examples/multimodal/speech_llm/conf/modular_audio_gpt_config_eval.yaml

```diff
@@ -0,0 +1,124 @@
+name: megatron_audio_gpt_eval
+
+trainer:
+  devices: 1
+  accelerator: gpu
+  num_nodes: 1
+  precision: bf16
+  logger: False # logger provided by exp_manager
+  enable_checkpointing: False
+  use_distributed_sampler: False
+  max_epochs: 9999
```

Comment:
minor: why not `-1`?

Reply:
we still want to use epoch based training and want the wandb to log epoch info, but megatron doesn't not work properly on LR scheduling with max_steps=-1, so we set a large max_steps to make sure LR is correct and also use max_epoch to log the epoch info